File: Chess/DataTypes/Table.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataTypes
{
    public class Table
    {
        public int GameNumber = 0;
        public RoundType RoundType = RoundType.Flop;
        public int Button = 0;
        public int CurrentPlayer = 1;
        public int RatePlayers = 0;


        public List<Player> Players = new List<Player>();
        public List<Card> Cards = new List<Card>();
        public CardDeck CardDeck = new CardDeck();
        public Bank Bank;
        public Table( int smallBlind )
        {
            Bank = new Bank(smallBlind);
        }


        public void SetSmallBlind(int smallBlind)
        {
            Bank.SetSmallBlind(smallBlind);
        }

        public void SetCards(List<Card> cards)
        {
            Cards = cards;
        }
    }
}

File: Chess/NodeChessServer/DataTypes/Table.js


const RoundType = require('./RoundType');
const CardDeck = require('./CardDeck');
const Bank = require('./Bank');

class Table {
    constructor(smallBlind) {
        this.gameNumber = 0;
        this.roundType = RoundType.flop;
        this.button = 0;
        this.currentPlayer = 1;
        this.ratePlayers = 0;
        this.players = [];
        this.cards = [];
        this.cardDeck = new CardDeck();
        this.bank = new Bank(smallBlind);
    }

    SetSmallBlind(smallBlind) {
        this.bank.SetSmallBlind(smallBlind);
    }

    SetCards(cards) {
        this.cards = cards;
    }
}
module.exports = Table;